### PR TITLE
Fixes all broken links besides auth & datastore

### PR DESCRIPTION
**Pull request by brene**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/cli/auth/groups.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: User groups
...
+You can create logical groups in Cognito User Pools and assign permissions to access resources in Amplify categories with the CLI, as well as define the relative precedence of one group to another. This can be useful for defining which users should be part of "Admins" vs "Editors", and if the users in a Group should be able to just write or write & read to a resource (AppSync, API Gateway, S3 bucket, etc). [You can also use these with `@auth` Static Groups in the GraphQL Transformer](/cli/graphql-transformer/directives.md#static-group-authorization). Precedence helps remove any ambiguity on permissions if a user is in multiple Groups.
```
Comment (nikhil-dabhade) [inline docs/cli/auth/groups.md:6]: Should this /cli/graphql-transformer/directives.md#static-group-authorization
be 
~/cli/graphql-transformer/directives.md#static-group-authorization

File: docs/lib/restapi/fragments/js/getting-started.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-The API category provides a solution for making HTTP requests to REST and GraphQL endpoints. For building GraphQL APIs please visit the [GraphQL guide](api-gql/intro)
+The API category provides a solution for making HTTP requests to REST and GraphQL endpoints. For building GraphQL APIs please visit the [GraphQL guide](~/lib/graphqlapi/getting-started.md)
 
 The REST API category can be used for creating signed requests against Amazon API Gateway when the API Gateway Authorization is set to `AWS_IAM`. 
 
-> Ensure you have [installed and configured the Amplify CLI and library](/js/start).
+> Ensure you have [installed and configured the Amplify CLI and library](~/cli/start/start.md).
```
Comment (nikhil-dabhade) [inline docs/lib/restapi/fragments/js/getting-started.md:5]: Should be ~/cli/start/install.md

Comment (brene) [reply docs/lib/restapi/fragments/js/getting-started.md:5]: fixed

File: docs/lib/xr/fragments/getting-started.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ Now click the *Download JSON configuration* button to download the scene configu
 
 ## Setup backend
 
-> Ensure you have [installed and configured the Amplify CLI and library](../../start/getting-started/installation.md)
+> Ensure you have [installed and configured the Amplify CLI and library](~/start/start.md)
```
Comment (nikhil-dabhade) [inline docs/lib/xr/fragments/getting-started.md:24]: Should this be ~/start/getting-started/installation.md ?

File: docs/sdk/analytics/fragments/android/getting-started.md [hunk 4]
```diff
@@ -12,8 +12,8 @@ Amazon Pinpoint is a fully managed AWS service that you can use to engage with y
 
 The Amplify CLI helps setup and configure Pinpoint within your application and connect with the AWS Mobile SDK.
 
-<b>Prerequisite:</b> [Install and configure the Amplify CLI](..)<br>
-<b>Recommendation:</b> [Complete the Getting Started guide](./start)
+<b>Prerequisite:</b> [Install and configure the Amplify CLI](~/cli/start/install.md)<br>
+<b>Recommendation:</b> [Complete the Getting Started guide](~/start/start.md)
```
Comment (nikhil-dabhade) [inline docs/sdk/analytics/fragments/android/getting-started.md:16]: Should this be ~/start/getting-started/installation.md ?

File: docs/lib/xr/fragments/getting-started.md (hunk 3 above)
Comment (brene) [reply docs/lib/xr/fragments/getting-started.md:24]: no, we want to redirect them to the getting-started home page, so they'll be able to choose their platform. The link works as is.

File: docs/sdk/api/fragments/ios/rest.md [hunk 5]
```diff
@@ -65,7 +65,7 @@ Find the resource name of your API by running `amplify status`. Copy your API cl
 
 ### IAM authorization
 
-To invoke an API Gateway endpoint from your application, import `AWSAPIGateway` and use the generated client class, model, and resource paths as in the below example with `YOUR_API_CLIENT_NAME` replaced from the previous section. For AWS IAM authorization use the `AWSMobileClient` as outlined in [the authentication section](/sdk/auth/how-it-works?platform=ios).
+To invoke an API Gateway endpoint from your application, import `AWSAPIGateway` and use the generated client class, model, and resource paths as in the below example with `YOUR_API_CLIENT_NAME` replaced from the previous section. For AWS IAM authorization use the `AWSMobileClient` as outlined in [the authentication section](~/sdk/auth/how-it-works).
```
Comment (nikhil-dabhade) [inline docs/sdk/api/fragments/ios/rest.md:68]: Should be ~/sdk/auth/how-it-works.md 
Missing .md

Comment (brene) [reply docs/sdk/api/fragments/ios/rest.md]: Fixed.

File: docs/sdk/storage/fragments/ios/getting-started.md [hunk 6]
```diff
@@ -57,7 +57,7 @@ If you had previously enabled user sign-in by running `amplify add auth` in your
 * Choose `Create groups`, and then choose `Finish`.
 
 ##### Lambda Triggers
-The Amplify CLI supports associating Lambda triggers for Amazon S3 and DynamoDB events. This can be useful for a use case where you want to invoke a Lambda function after a create or update operation on a DynamoDB table managed by the CLI. [Read More](/cli/storage)
+The Amplify CLI supports associating Lambda triggers for Amazon S3 and DynamoDB events. This can be useful for a use case where you want to invoke a Lambda function after a create or update operation on a DynamoDB table managed by the CLI. [Read More](/cli/storage/storage.md)
```
Comment (nikhil-dabhade) [inline docs/sdk/storage/fragments/ios/getting-started.md:60]: Should be ~/cli/storage/storage.md

**Pull request merged**